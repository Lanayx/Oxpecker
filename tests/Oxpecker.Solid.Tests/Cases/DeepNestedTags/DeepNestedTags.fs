module Oxpecker.Solid.Tests.Cases.DeepNestedTags

open Oxpecker.Solid

[<SolidComponent>]
let Test () =
    html(xmlns = "http://www.w3.org/1999/xhtml") {
        body() {
            div(id="outer") {
                div(id="inner") {
                    h1() { "Hello!" }
                }
            }
        }
    }

