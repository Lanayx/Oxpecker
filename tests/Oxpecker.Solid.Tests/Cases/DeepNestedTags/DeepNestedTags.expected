
export function Test() {
    return <html xmlns="http://www.w3.org/1999/xhtml">
        <body>
            <div id="outer">
                <div id="inner">
                    <h1>
                        Hello!
                    </h1>
                </div>
            </div>
        </body>
    </html>;
}

